deployment/environment/devenv/don.go

```diff
@@ -168,11 +168,19 @@ func NewNode(nodeInfo NodeInfo) (*Node, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to create node rest client: %w", err)
 	}
+	labels := []*ptypes.Label{}
+	if nodeInfo.IsBootstrap {
+		labels = append(labels, &ptypes.Label{
+			Key:   NodeLabelKeyType,
+			Value: ptr(NodeLabelValueBootstrap),
+		})
+	}
```

Comment:
why is this needed? it should send all labels from nodeinfo instead, please refer to this - https://github.com/smartcontractkit/chainlink/blob/develop/deployment/environment/devenv/don.go#L165-L171